requirements.txt

```diff
@@ -2,4 +2,6 @@ peakrdl
 mako
 pre-commit
 numpy
-cocotb
\ No newline at end of file
+cocotb
+cocotb-bus
+pytests
```

Comment:
This should be `pytest`?

Reply:
Fixed, there were a few other things added too:
- in `regressions/test.py` corrected handling of environment variables if they werent set by the user
- Corrected active/inactive reset value in `regressions/testbench.reset()` (previously hard coded despite the logic being there to deduce active high/low form the reg map)